packages/pigeon/pigeons/core_tests.dart

```diff
@@ -18,6 +18,13 @@ enum AnotherEnum {
   justInCase,
 }
 
+// This exists to show that unused data classes still generate.
+class UnusedClass {
+  UnusedClass({this.aField});
+
+  Object? aField;
+}
+
 class SimpleClass {
```

Comment:
Based on the output diffs, it looks like this class wasn't being generated before. That suggests that it should be unused, and thus removed.

Reply:
Or it should be used to validate that unused classes are generated? https://github.com/flutter/packages/pull/7529#pullrequestreview-2267160848